### PR TITLE
fix broken links; remove temp discussion

### PR DESCRIPTION
**Pull request by TomConner**
https://ga4gh.github.io/data-security/check-links/

### CODE REVIEW

File: AAI/FAQ.md [hunk 1]
```diff
@@ -459,9 +459,7 @@ of the AAI standard regarding the presence of JKUs.
 
 {% hr2 %}
 
-## Threat Analysis
-
-### What is the danger of using a fully scoped (or audience-less) token in a multi-node workflow
+### What is the limitation of using a fully scoped (or audience-less) token in a multi-node workflow?
```
Comment (kwrodarmer) [inline AAI/FAQ.md:462]: I suggest the heading be changed from "Threat Analysis" to "Known Limitations" and this line be changed to "What are the pros and cons of using a fully scoped (or audience-less) token in a multi-node workflow?"

**Review by kwrodarmer — DISMISSED** [review]
I suggest changing the "Threat Analysis" heading be changed from "Threat Analysis" to "Known Limitations" rather than it being removed from FAQ, and the question line be changed to "What are the pros and cons of using a fully scoped (or audience-less) token in a multi-node workflow?"

**Review by martin-kuba — APPROVED** [review]

File: AAI/FAQ.md (hunk 1 above)
Comment (martin-kuba) [reply AAI/FAQ.md:462]: I agree, "Known Limitations" is more accurate heading for this section.

Comment (TomConner) [reply AAI/FAQ.md]: I have made the change, thanks!

**Comment by TomConner** [comment]
> I suggest changing the "Threat Analysis" heading be changed from "Threat Analysis" to "Known Limitations" rather than it being removed from FAQ, and the question line be changed to "What are the pros and cons of using a fully scoped (or audience-less) token in a multi-node workflow?"

Change made.

**Pull request merged**